_layouts/default.html

```diff
@@ -21,7 +21,14 @@
     <meta name="description" content="Sample illustrating the use of {{ page.feature_name }}.">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% if page.title %}{{ page.title }}{% else %}{{ page.feature_name }} Sample{% endif %}</title>
-    {% if page.https_redirect %}{% include https_redirect.html %}{% endif %}
+    <script>
+      // If we're running on a real web server (as opposed to localhost on a custom port,
+      // which is whitelisted), then change the protocol to HTTPS.
+      // See https://goo.gl/lq4gCo for an explanation as to why this is needed for some features.
+      if ((!location.port || location.port == "80") && location.protocol != 'https:') {
```

Comment:
I'm following @jakearchibald's lead here, e.g.: https://github.com/jakearchibald/svgomg/blob/02f4dd3666aee701792420cdf52b8f7819fc05f8/src/index.html#L9

Are you suggesting this to avoid a redirect when the page is loaded with via `file:`? Or is there some other benefit?